### PR TITLE
Us 36 Add candidates list 

### PR DESCRIPTION
**Pull request by mateuszs-kainos**

### CODE REVIEW

File: src/main/java/resources/PersonResource.java [hunk 1]
```diff
@@ -115,4 +115,20 @@ public Person updatePerson(Person person) {
         personDao.update(person);
         return person;
     }
+
+    @GET
+    @Path("/recruiter")
+    @UnitOfWork
+    public Response getRecruiter() {
```
Comment (msolecka-kainos) [inline src/main/java/resources/PersonResource.java]: This method return list of recruiters, so we could rename method and path to  fetchRecruiters  and /all/recruiter like above fetchPersons ?

File: src/main/resources/assets/html/partials/recruiter/tdpr-recruiter-candidates.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                        | <a class="cursor-pointer" ng-click="showPopupForEdit(candidate)">edit</a>
+                        | <a class="cursor-pointer" ui-sref='tdpr.recruiter.home({candidateId: candidate.id, candidate: candidate})'>schedule</a>
+                    </td>
+
+                </tbody>
+            </table>
+        </div>
+    </div>
+
+    <a id="via-service" href="" ng-click="showPopupForAdd()">Add new candidate</a> |
```
Comment (msolecka-kainos) [inline src/main/resources/assets/html/partials/recruiter/tdpr-recruiter-candidates.html]: Is empty href needed?

File: src/main/java/dao/CandidateDao.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    @Inject
+    public CandidateDao(SessionFactory sessionFactory){
+        super(sessionFactory);
+    }
+
+    public List<Candidate> findAll(){
+        return namedQuery("Candidate.findAll").list();
+    }
+
+    public Optional<Candidate> findById(Long id){
```
Comment (rlewan) [inline src/main/java/dao/CandidateDao.java:22]: 👍

File: src/main/java/resources/CandidateResource.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    @PUT
+    @Path("/update")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @UnitOfWork
+    public Candidate update(Candidate candidate){
+        candidateDao.update(candidate);
+        return candidate;
+    }
+
+    @GET
```
Comment (rlewan) [inline src/main/java/resources/CandidateResource.java]: `GET` operations should not modify data -- please use `PUT` or `DELETE` instead.

Comment (jraniszewski) [reply src/main/java/resources/CandidateResource.java]: Fixed :)

**Pull request merged**